api/nakadi-event-bus-api.yaml

```diff
@@ -1083,6 +1235,60 @@ definitions:
         items:
           $ref: '#/definitions/Event'
 
+  Subscription:
+    description: |
+      Subscription is a high level consumption unit. Subscriptions allow applications to easily scale the number of
+      clients by managing consumed event offsets and distributing load between instances.
+      The key properties that identify subscription are 'owning_application', 'event_types' and 'use_case'.
+      It's not possible to have two different subscriptions with these properties being the same.
+    properties:
+      id:
+        type: string
+        description: |
+          Id of subscription that was created. Is generated by Nakadi, should not be specified when creating
+          subscription.
+      created_at:
```

Comment:
Long time passed.. but this is still in PR, so I'll add it. I think that we need to split Subscription into 2 classes:
 - Subscripiton
 - StoredSubscription

StoredSubscription will hold all the values from Subscription and additionaly it will hold id and created_at field. From my point of view it will simplify development.